docs/source/markdown/podman-run.1.md

```diff
@@ -1011,8 +1011,11 @@ When set to **true**, Podman will allocate a pseudo-tty and attach to the standa
 input of the container. This can be used, for example, to run a throwaway
 interactive shell. The default is **false**.
 
-**NOTE**: The **-t** option is incompatible with a redirection of the Podman client
-standard input.
+**NOTE**: The --tty flag prevents redirection of standard output (combines STDOUT and STDERR, can insert control characters, can hang pipes. It should only be used when run interactivley in a terminal. When feeding input to Podman, use -i only, not -it.
```

Comment:
missing an end paren,  Maybe?
```suggestion
**NOTE**: The --tty flag prevents redirection of standard output.  It combines STDOUT and STDERR, it can insert control characters, and it can hang pipes. This option should only be used when run interactively in a terminal. When feeding input to Podman, use -i only, not -it.
```